### PR TITLE
Add support for deploy percentage

### PR DESCRIPTION
**Pull request by remdash**
Following discussion in https://github.com/fregante/chrome-webstore-upload/pull/91#issuecomment-2154102315 here is an update of the CLI

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -42,6 +42,7 @@ $ chrome-webstore-upload --help
     --refresh-token     OAuth2 Refresh Token (environment variable REFRESH_TOKEN)
     --auto-publish      Can be used with the "upload" command (deprecated, use `chrome-webstore-upload` without a command instead)
     --trusted-testers   Can be used with the "publish" command
+    --deploy-percentage Can be used with the "publish" command
```
[the remark this answers is not in the document]
Comment (fregante) [reply readme.md]: Also in the actual CLI

File: test/config.js [hunk 2]
```diff
@@ -58,6 +58,12 @@ test('Auto Publish', async t => {
     t.true(config.autoPublish);
 });
 
+test('Publish with deploy percentage', async t => {
+    const config = await createConfig(null, { deployPercentage: 5 });
+
+    t.is(config.deployPercentage, 5);
+});
```
[the remark this answers is not in the document]
Comment (remdash) [reply test/config.js:65]: Good point. I've added one.

File: readme.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (remdash) [reply readme.md]: It's fixed there too.

File: test/config.js [hunk 3]
```diff
@@ -58,6 +58,18 @@ test('Auto Publish', async t => {
...
+
+test('Publish without deploy percentage', async t => {
+    const config = await createConfig(null, {});
+
+    t.falsy(config.deployPercentage);
```
[the remark this answers is not in the document]
Comment (remdash) [reply test/config.js]: Indeed, I've fixed it. Thanks

**Comment by fregante** [comment]
3.3.0 is out 🙌

**Pull request merged**